tardis/model.py

```diff
@@ -207,7 +207,7 @@ def calculate_j_blues(self, init_detailed_j_blues=False):
             logger.info('Calculating J_blues for radiate_rates_type=detailed')
 
             self.j_blues = pd.DataFrame(
-                self.j_blue_estimators.transpose() *
+                self.j_blue_estimators *
                 self.j_blues_norm_factor.value,
```

Comment:
After getting informed more about coding styles I have to admit indenting the next line would look better and improve readability.

We should definitely write some code guidelines for the documentation and every time we encounter something not handled yet add it.